guides/source/upgrading_ruby_on_rails.md

```diff
@@ -387,7 +393,7 @@ by the aforementioned bug, this configuration should be enabled:
 config.active_record.encryption.support_sha1_for_non_deterministic_encryption = true
 ```
 
-If you are working with encrypted data, please carefully review the above.
+**If you are working with encrypted data, please carefully review the above.**
```

Comment:
Instead of making this text bold you can highlight it by using [NOTE or WARNING](https://edgeguides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#notes-tips-and-warnings)
```suggestion
WARNING: If you are working with encrypted data, please carefully review the above
```